spec/requests/api/v1/users/show_spec.rb

```diff
@@ -27,7 +27,7 @@
 
   context 'when an unexpected error occurs' do
     it 'returns status 500 internal server error' do
-      allow(Rails).to receive(:env).and_return 'production'.inquiry
+      allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new('production'))
```

Comment:
Why?

Reply:
Because yes. 

Honestly I'm not sure what this test is for, why should we test our app responds with a 500 error code? I'm more inclined into removing this test altogether. 

Any suggestions?

Reply:
Discussed over slack: the server should not be rescuing from `Exception`